app/src/androidTest/java/ch/epfl/sweng/rps/MainActivityTest.kt

```diff
@@ -1,27 +1,70 @@
 package ch.epfl.sweng.rps
 
-import androidx.test.espresso.Espresso
-import androidx.test.espresso.action.ViewActions
+
+import android.view.Gravity
+import androidx.test.InstrumentationRegistry.getTargetContext
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu
 import androidx.test.espresso.action.ViewActions.click
 import androidx.test.espresso.assertion.ViewAssertions.matches
-import androidx.test.espresso.matcher.ViewMatchers
-import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.espresso.contrib.DrawerActions
+import androidx.test.espresso.contrib.DrawerMatchers.isClosed
+import androidx.test.espresso.contrib.NavigationViewActions
+import androidx.test.espresso.matcher.ViewMatchers.*
 import androidx.test.ext.junit.rules.ActivityScenarioRule
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.filters.SmallTest
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
+
 @RunWith(AndroidJUnit4::class)
 @SmallTest
 class MainActivityTest {
+
     @get:Rule
     val testRule = ActivityScenarioRule(MainActivity::class.java)
 
     @Test
-    fun pressingGreetingButton() {
-        val name = "John"
+    fun clickOnStatistics_OpensStatisticFragment() {
```

Comment:
Function name like here can be less repetitive and follow one naming convention (camelCase). Besides that LGTM